src-tool/Pact/Analyze/Model/Text.hs

```diff
@@ -24,6 +24,7 @@ import qualified Data.SBV.Internals         as SBVI
 import           Data.Text                  (Text)
 import qualified Data.Text                  as T
 import           GHC.Natural                (Natural)
+import qualified Data.Decimal as Pact
```

Comment:
Try and keep with the style of the module as tedious as it is until we come up with a formatting policy